support/docker/dev/janitor.json

```diff
@@ -19,13 +19,17 @@
       "proxy": "https"
     },
     "9000": {
-      "label": "PeerTube",
+      "label": "PeerTube API",
+      "proxy": "https"
+    },
+    "3000": {
```

Comment:
Super-nit: Please sort ports from smallest to largest (i.e. `"3000"` before `"9000"`).